HousingSearchApi.Tests/V1/E2ETests/Fixtures/ProcessFixture.cs

```diff
@@ -21,7 +21,7 @@ public class ProcessFixture : BaseFixture
         public const string INDEX = "processes";
         private static readonly Fixture _fixture = new Fixture();
 
-        private static PatchAssignment _patchAssignment = _fixture.Create<PatchAssignment>();
+        public static PatchAssignment _patchAssignment = _fixture.Create<PatchAssignment>();
```

Comment:
```suggestion
        public static PatchAssignment PatchAssignment = _fixture.Create<PatchAssignment>();
```

To match naming conventions